libtelemetry/src/main/java/com/mapbox/android/telemetry/MapboxTelemetry.java

```diff
@@ -55,12 +55,14 @@
   private boolean isServiceBound = false;
   private PermissionCheckRunnable permissionCheckRunnable = null;
   private CopyOnWriteArraySet<TelemetryListener> telemetryListeners = null;
+  private CertificateBlacklist certificateBlacklist;
   static Context applicationContext = null;
 
   public MapboxTelemetry(Context context, String accessToken, String userAgent) {
     initializeContext(context);
     initializeQueue();
     checkRequiredParameters(accessToken, userAgent);
+    checkBlacklist();
```

Comment:
Wouldn't it make more sense to pass context and accessToken as parameters to this method to be consistent with other initialization methods?